index.d.ts

```diff
@@ -40,7 +40,7 @@ declare const pTimeout: {
 	): ClearablePromise<ValueType>;
 
 	/**
-	Timeout a promise after a specified amount of time.
+	Timeout a promise after a specified amount of time. It returns a decorated promise having `.clear()` method to be able to clear timeout.
```

Comment:
This is not in the same location as the readme. I think it could also be better written.

Reply:
I refactored the docs. But really don't know how it could be better :)

Reply:
You didn't fix it in both places.

Reply:
This addition should be removed.